asymmetric_manipulation/Controllers control_mode
asymmetric_manipulation/Inputs input_mode
bool use_limits
asymmetric_manipulation/WorkspaceLimits limits
float64 max_time
float64 alpha
bool dynamic_alpha
bool symmetric_secundary_task
bool use_asymmetric_l_only
---
---
float64 dt
float64 curr_alpha
float64 manip1
float64 manip2
float64 manip_joint
float64 absolute_angle
float64 obj1_angle
float64 obj2_angle
float64 joint_space_norm
float64 relative_error_norm
float64 relative_error_angle
geometry_msgs/Twist v1
geometry_msgs/Twist v2
geometry_msgs/Twist va
geometry_msgs/Point relative_error
geometry_msgs/Point absolute_position
geometry_msgs/Point p1
geometry_msgs/Point p2
